bsp/freedom-e310-arty/metal.h

```diff
@@ -861,11 +861,6 @@ struct __metal_driver_cpu *__metal_cpu_table[] = {
 
 #define __METAL_DT_INTERRUPT_CONTROLLER_C000000_HANDLE (&__metal_dt_interrupt_controller_c000000.controller)
 
-/* From local_external_interrupts_0 */
-#define __METAL_DT_SIFIVE_LOCAL_EXINTR0_HANDLE (&__metal_dt_local_external_interrupts_0.irc)
```

Comment:
Why was these get deleted?

Reply:
I removed them in a recent fix for freedom-devicetree-tools. They weren't being used by anything, were private API, and were generating compile warnings because multiple were getting generated which conflicted with each other.

Reply:
Oh OK. Just that we have not regenerate/update the BSP files in Freedom-e-sdk. Thanks